docs/blog/2018-06-15-why-we-built-the-site-showcase/index.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Why We Created the Site Showcase
+date: 2018-06-15
+author: Shannon Soper
+tags: [""]
+---
+ 
+We recently finished the first phase of the [Site Showcase](https://next.gatsbyjs.org/showcase/), which is a searchable, filterable library of sites built with Gatsby!
+
+![Featured Sites](featured-sites.png)
+
+![Filtered Sites](filtered-sites.png)
+
+## Submitting your site
+
+If you’d like to submit a site to the showcase, please follow the [instructions in this document](https://github.com/gatsbyjs/gatsby/blob/v2/docs/docs/site-showcase-submissions.md).
+ 
+# Why build a site showcase?
+
+We built the showcase to make it easier for you to find the source code for Gatsby sites and easier to show off what Gatsby can do to potential clients / coworkers.
+
+## Source code
+
+Many people learn how to build things by looking at src code for other sites. As you browse through the site showcase, look for the GitHub icon on each site’s detail page which links you to the source code (not all sites will have it):
```

Comment:
Sometimes, examples are worth 1000 words. That's why we added a  GitHub icon next to sites with open source code.

Reply:
(replace sentences with ^)